stable/kong/templates/service-kong-admin.yaml

```diff
@@ -13,9 +13,17 @@ metadata:
     heritage: "{{ .Release.Service }}"
 spec:
   type: {{ .Values.admin.type }}
-  {{- if and (eq .Values.admin.type "LoadBalancer") .Values.admin.loadBalancerIP }}
+  {{- if eq .Values.admin.type "LoadBalancer" }}
+  {{- if .Values.admin.loadBalancerIP }}
   loadBalancerIP: {{ .Values.admin.loadBalancerIP }}
   {{- end }}
+  {{- if .Values.admin.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
```

Comment:
I'm new to the go templating language so if there's a better way to format this I'm totally open to suggestions.

Reply:
This looks good. The last one indeed gave an error as it was expanding as an array directly. Thanks for fixing this!